checkup.go

```diff
@@ -357,6 +357,8 @@ type Checker interface {
 // Storage can store results.
 type Storage interface {
 	Store([]Result) error
+	Fetch(string) ([]Result, error)
+	GetIndex() (map[string]int64, error)
```

Comment:
Hm, maybe instead of adding methods to this interface, make a new interface called `StorageLoader` or something (the name is up for discussion if you want) that has these two methods.